Backend/databaseInteraction/databaseInterface.js

```diff
@@ -1,10 +1,11 @@
-const {models} = require('../sequelizeSetup/sequalizeConstructor')
+const {models} = require('../sequelizeSetup/sequalizeConstructor');
 
 /**
  * Finds a user from a given username/password
+ *
  * @param {string} email the email address of the user trying to be found
  * @param {string} password the password of the user trying to be found
- * @returns the user object, if found, otherwise false
+ * @returns {boolean | object} the user object, if found, otherwise false
```

Comment:
I don't know if this is a common thing to do in JS, but I feel like having a function return multiple types is not great. I guess it doesn't really matter if you're the only one using it tho.

Reply:
That's fair, changed to return an object or null which is written as `object?` in js. Does that work?

Reply:
Yeah sounds good